torch/csrc/jit/runtime/static/impl.cpp

```diff
@@ -251,7 +251,7 @@ LivenessInformation GetLivenessInformation(
 }
 
 // Implementation specific pruning of values
-// from "optimzable" set.  GetLivenessInformation and FindShared
+// from "optimzable" set.  GetLivenessInformation and FindSameStorageValues
```

Comment:
Might even make sense to call this "GenerateSameStorageValues" as all the decision making happens here

Reply:
Done